docs/devel-quick-start.md

```diff
@@ -0,0 +1,191 @@
+   ```
+
+That's all! You are ready to submit your first pull request!
+
+## Setting up the `drenv` tool
+
+*Ramen* uses the `drenv` tool to build a development environment
+quickly. Please follow the
+[Setup instructions](https://github.com/RamenDR/ramen/tree/main/test#setup)
+to set up the `drenv` tool.
+
+## Starting the test environment
+
+*Ramen* supports many configurations, but the only development
+environment available now is the `regional-dr.yaml`. This environment
+simulates a Regional DR setup, when the managed clusters are located in
+different zones (e.g. US, Europe) and storage is replicated
+asynchronously. This environment includes 3 clusters - a hub cluster
+(`hub`), and 2 managed clusters (`dr1`, `dr2`) using `Ceph` storage.
+
+To start the environment, run:
+
+```
+drenv start regional-dr.yaml
+```
+
+Starting takes at least 5 minutes, depending on your machine and
+internet connection.
+
+You can inspect the environment using kubectl, for example:
+
+```
+kubectl get deploy -A --context hub
+kubectl get deploy -A --context dr1
+kubectl get deploy -A --context dr2
+```
+
+## Building the Ramen operator
+
+1. Setup the environment variables for the Makefile
+
+   ```
+   export IMAGE_REPOSITORY=my-quay-user
+   export IMAGE_TAG=canary
+   ```
+
+1. Build the *Ramen* operator container:
+
+   ```
+   make docker-build
+   ```
+
+   This builds the image `quay.io/my-quay-user/ramen-operator:canary`
+
+1. Pushing the `ramen-operator` image to the clusters
+
+   You can push the container image to your private quay repo, or load
+   them into the clusters.
+
+    - Loading the `ramen-operator` image into the clusters
+
+      ```
+      podman save quay.io/nirsof/ramen-operator:canary \
+          -o /tmp/ramen-operator.tar
+
+      for p in hub dr1 dr2; do
+          minikube -p $p image load /tmp/ramen-operator.tar
+      done
+
+      rm /tmp/ramen-operator.tar
+      ```
+
+    - Pushing the `ramen-operator` image to your private repo
+
+      ```
+      make docker-push
+      ```
+
+      You may need to login first using `podman login quay.io`.
+
+1. Deploying to hub and cluster
+
+   ```
+   kubectl config use-context hub
+   make deploy-hub
+
+   for p in dr1 dr2; do
+       kubectl config use-context $p
+       make deploy-dr-cluster
+   done
+   ```
+
+1. Disable *volsync* and *kube object protection*
+
+   Ramen test environment does not include yet `volsync` and `velero`, so we
+   need to disable some options in *Ramen* config on the dr cluster.
+
+   Edit `ramen-dr-cluster-operator-config` configmap on the dr clusters:
+
+   ```
+   for p in dr1 dr2; do
+       kubectl edit configmap ramen-dr-cluster-operator-config -n ramen-system --context $p
+   done
+   ```
+
+   Add these lines to the configmap:
+
+   ```
+    volSync:
+      disabled: true
+    kubeObjectProtection:
+      disabled: true
+   ```
+
+1. Restart `ramen-operator`
+
+   *Ramen* does not watch yet changes to the configmap, so we need to
```

Comment:
Ramen watches for config map changes, but does not restart its pods (or rather just aborts and let k8s restart the pod), when the above options are changed. These options change what resources we watch for in the reconcilers and that requires a restart of the operator.

IOW, reword so that we state the cause of failure more accurately.

Reply:
Thanks, will change to

    Reman does not apply yet changes in configmap automatically, ...

I assume that we want to add automatic reconfiguration on changes.

Reply:
Fixed in current version